ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -46,7 +46,7 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
     } | null>(null);
 
     function onChangeLifecycleStage(lifecycleStage: LifecycleStage, isChecked: boolean) {
-        if (values.id) {
+        if (values.policySections.length > 0 && values.policySections[0].policyGroups.length > 0) {
```

Comment:
One edge case -- if a user applies multiple policy sections and then deletes the rules from the first section this condition will not trigger.

Reply:
Shoot, you're right. Fixed.